### ISSUE TITLE
Port "This Week in Wagtail" to wagtail-newsletter

### ISSUE THREAD

**Issue by mgax** [wagtail/wagtail.io#481]
The newsletter used to be content-managed in wagtail.org until July 2023 (the latest page is https://wagtail.org/this-week-in-wagtail/issue-136/). Content display issues on Outlook prompted a move to a content template in Mailchimp.

Since https://github.com/wagtail/wagtail-newsletter is released, and it [comes with MJML support](https://wagtail-newsletter.readthedocs.io/en/stable/customization.html#newsletter-templates), we can reimplement the current Mailchimp template as MJML, and move back to Wagtail-managed newsletters.

**Comment by mgax** [comment]
I think a good plan is to:
1. Set up a local dev environment for this repo.
2. Set up `wagtail-newsletter`.
3. Recreate a few TWiW episodes. The latest is [15 November 2024 – Help us improve StreamField blocks](https://mailchi.mp/wagtail/twiw-11032065?e=d7f7e52ea2). @vossisboss can you point to a few other examples that contain varied content, so we cover the breadth of content features currently in use?

**Comment by vossisboss** [comment]
Hi @mgax. This sounds like a solid plan to me. Honestly, the last few issues use varied content in them. Here are some links:

https://mailchi.mp/wagtail/twiw-11032065?e=[UNIQID]
https://mailchi.mp/wagtail/twiw-11031731?e=[UNIQID]
https://mailchi.mp/wagtail/twiw-11031437?e=[UNIQID]
https://mailchi.mp/wagtail/twiw-10706474?e=[UNIQID]
https://mailchi.mp/wagtail/twiw-10706442?e=[UNIQID]

I can give you access to MailChimp, if needed.

One ask for the template.  MailChimp doesn't offer the ability to add BlueSky or Mastodon links without custom code in the social media bar and I never was able to Outlook-proof that code. It kept breaking in Litmus tests and so I abandoned it and reverted to using the default. We don't use Twitter anymore, so we'll want to include Mastodon, Bluesky, YouTube, and LinkedIn.

We still have a Litmus account at Torchbox it looks like. The login is in Bitwarden. Whatever template you set up, we should make sure to run tests with Litmus too because the last time we had a custom template, Outlook kept breaking it.

**Comment by vossisboss** [comment]
@mgax It just occurred to me that you already created a template for some of the demos you presented. Is there a reason that template code could not be used for doing the actually update? At the very least, I think it would be helpful for someone taking this on to have access to your demo. It might speed things up.

**Comment by mgax** [comment]
@vossisboss good point! The template is actually part of the demo project that is part of the wagtail-newsletter repo: https://github.com/wagtail/wagtail-newsletter/blob/main/demo/templates/demo/article_page_newsletter.html.

**Comment by vossisboss** [comment]
I think those sound pretty good @mgax. I would appreciate being able to use all heading options within the Rich text blocks as well. Please don't limit those to just heading blocks. It's a better writing experience to be able to do that.

The only other thing can think of off the top of my head that should be updated in the template are our social media account buttons. MailChimp didn't support adding Bluesky or Mastodon. I would like use to include links for LinkedIn, Bluesky, Mastodon, and YouTube.

If I think of anything else, I'll let you know.
